client.go

```diff
@@ -293,7 +299,7 @@ func (s *Client) read() (messageType int, p []byte, err error) {
 	if s == nil {
 		return 0, nil, errNotInit
 	}
-
+	s.conn.SetReadDeadline(time.Now().Add(s.config.ReadTimeOut))
```

Comment:
How will it help to distinguish normal idling from broken connect? Ping-pong is not always necessary requirement.

Reply:
Must be:
```
conn.SetPingHandler(func(appData string) error {
				conn.WriteMessage(stubborn.PongMessage, []byte(appData))
				return conn.SetReadDeadline(time.Now().Add(readTimeOut))
			})
```